src/main/java/com/rtb/projectmanagementtool/project/ProjectController.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * ProjectController.java - this file implements ProjectController, which controls the logic of
+ * working with Project objects.
+ */
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.HashSet;
+
+public class ProjectController {
+  private DatastoreService datastore;
```

Comment:
Probably would be better to pass this from outside, rather than having a local instance here.

For most database engines, having a "database client" or "database connection" would mean it'll hold on to the connection until the process is done.  Typically you'd want to limit the number of connections you have open to save on resources, so it's usually advisable to have 1 connection and pass it around (via params to db consumers), rather than having one connection per consumer.